dnd.sty

```diff
@@ -153,6 +153,15 @@
   \dndline%
 }
 
+\renewcommand{\LettrineFontHook}{\dnd@DropCapFont}
+% Usage: DndDropCapLine[<number of descender lines>]{<first letter>}{<first line>}<rest of paragraph...>
+%    It takes trial and error to get the 2nd argument to align with
+%    the linebreak. Lettrine package will not play nicely with \FirstLine
+%    The initial argument is optional and is used for letters with lower descender (default is 0)
+%    For example a capital Y needs an extra line
+\NewDocumentCommand{\DndDropCapLine}{ O{0} m m }%
+{\lettrine[lines=4,depth=#1,findent=0.5em,nindent=0em]{#2}{}\textsc{#3}}
```

Comment:
`\lettrine` already puts the third argument in small caps, so we can remove the `\textsc`. Also, with a small change, we can expose all the power of `\lettrine` while setting the defaults we want. This is especially useful for finicky fonts like ArtNouveauC. `\wordsep` may look more natural for the gap on the right of the drop cap.

```
\NewDocumentCommand{\DndDropCapLine}{ O{} m m }%
{\lettrine[lines=4,depth=0,findent=\wordsep,nindent=0em,#1]{#2}{#3}}
```